index.js

```diff
@@ -22,14 +22,33 @@ const streamersDaCaverna = JSON.parse(
     flag: "r",
   })
 );
+const nomeStreamers = [];
 const streamersOn = [];
 const tempoDivulgacao = 600000;
 const tempoAtualizacao = 500000;
 
+streamersDaCaverna.forEach((streamer) => {
+  nomeStreamers.push(streamer.name);
+});
+
 function recivedMessage(target, context, msg, bot) {
   if (bot) {
     return;
   }
+
+  let { username } = context;
+  let isMod = context.mod;
+  let command = msg.split(" ");
+
+  if (
+    (isMod || streamersDaCaverna.includes(username)) &&
```

Comment:
Isso aqui permite que um streamer da caverna use o comando em um canal de outra pessoa, mesmo não sendo moderador no canal. Se a ideia é essa mesma, está correto. Se o comando for acessível somente pra moderadores e pro streamer daquele canal, pode usar:

```js
if(isMod || (context.badges && 'broadcaster' in context.badges))
```

Reply:
Só precisa confirmar pq esse `context.badges` vem todas as badges do usuário, inclusive a de broadcaster.
A documentação do TMI é muito ruim, mas **acho** que a condicional é essa...